api/dp-openapi.yaml

```diff
@@ -1020,6 +1189,69 @@ components:
       items:
         type: string
 
+    proof_page_text:
+      type: object
+      properties:
+        text:
+          description: the page text from current round.
+          type: string
+
+    proof_page_data:
+      type: object
+      properties:
+        text:
+          description: the page text. If page state is 'out' from OCR or previous round, else from current round
+          type: string
+        pagestate:
+          description: the page state, for checking continuity. Either out or temp
+          type: string
+        saved:
+          description: true if the page state is 'temp'
+          type: boolean
+        pagename:
+          description: the filename of the page with extension
+          type: string
+        image_url:
+          description: url of the image of the page
+          type: string
+        pageNum:
+          description: page "number" without extension
+          type: string
+        roundInfoArray:
+          description: array of names of proofreaders in earlier rounds
+          type: array
+          items:
+            type: string
+
+    page_text_state:
+      type: object
+      properties:
+        text:
+          description: the page text, current or original.
+          type: string
+        pagestate:
+          description: the page state, it is now temp
+          type: string
+        saved:
```

Comment:
Yes, this needs more explanation. Although we were at one time considering removing the page_out_state, it could be useful to keep it in case it could be used in the future in connection with checking out several pages to distinguish the ones that haven't yet been looked at. This version stays close to the original and, when checking out a page, does not change the current page text.
If we have a button for 'Revert to last save' then it is disabled if the page hasn't yet been saved (as it works now). That is the only purpose of this 'saved' value.